src/SkynetServer/Verification/VerificationMailer.cs

```diff
@@ -0,0 +1,37 @@
+﻿using MailKit.Net.Smtp;
+using Microsoft.Extensions.Configuration;
+using MimeKit;
+using SkynetServer.Configuration;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SkynetServer.Verification
+{
+    internal class VerificationMailer
+    {
+        public async Task SendMailAsync(string address, string token)
+        {
+            MailConfig config = Program.Configuration.Get<MailConfig>();
+
+            var message = new MimeMessage();
+            message.From.Add(new MailboxAddress(config.SenderName, config.SenderAddress));
+            message.To.Add(new MailboxAddress(address));
+            message.Subject = "Skynet Messenger mail address verification";
```

Comment:
```suggestion
            message.Subject = "Confirm your email address";
```
We already have the name "Skynet Team" in the sender name, so "Skynet Messenger" can be omitted from the subject.